client/scripts.js

```diff
@@ -1,62 +1,69 @@
 // This token will be received from the server upon joining
 // Use it to identify yourself when sending messages
-var serverToken = "";
+var serverToken = '';
 // Displayed name of current user
-var username = "test" + Date.now().valueOf().toString();
+var username = 'test' + Date.now().valueOf().toString();
 
 var chatWindow = document.getElementById('chatWindow');
 var messageWindow = document.getElementById('');
-var btnChatMessageSend = document.getElementById("btnMessageSend");
-var textAreaChatMessage = document.getElementById("textChatMessageField");
+var btnChatMessageSend = document.getElementById('btnMessageSend');
+var textAreaChatMessage = document.getElementById('textChatMessageField');
 
 // Create websocket connection
-const socket = new WebSocket("ws://localhost:8080", "chat");
+const socket = new WebSocket('ws://localhost:8080', 'chat');
 
 // Connection opened
-socket.addEventListener("open", function(event) {
-    console.log("Opened socket to the chat server, connecting to chat...");
-    connectToServer(socket);
+socket.addEventListener('open', function(event) {
```

Comment:
Comment out this whole event listener to disabled automatic connection to the chat.